bigbluebutton-html5/imports/ui/components/actions-bar/styles.scss

```diff
@@ -30,3 +30,7 @@
 .invisible {
   visibility: hidden;
 }
+
+.isTalking {
+  text-shadow: 0 0 .35rem #FFF;
```

Comment:
It could add confusion though because often when a whole button is highlighted it means that there's some importance to the button at that time or it needs immediate attention, but in our case the glow is there to give more information and doesn't require use interaction.